input/cql/Status.cql

```diff
@@ -16,7 +16,12 @@ context Patient
 //This library contains functions that are based on QDM 5.6 to QICore 4.1.1 March 2023 (https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Authoring-Patterns---QICore-v4.1.1). The functions may appear similar to some QICoreCommon functions but different in that they have constraints that are relevant for measures authored by NCQA.
 
 //Assessment, Performed
-define fluent function isAssessmentPerformed(Obs List<ObservationScreeningAssessment>):
+define fluent function isAssessmentPerformed(Obs List<Choice<"SimpleObservation", 
+"LaboratoryResultObservation", 
+"ObservationClinicalResult",
+"ObservationScreeningAssessment",
+"NonPatientObservation", 
+"ObservationCancelled">>):
```

Comment:
According to the [mapping](https://hl7.org/fhir/us/qicore/STU6/qdm-to-qicore.html#assessment-performed), this should be mapped to only QI core SimpleObservation and ObservationScreening Assessment